extensions/smallrye-stork/runtime/src/main/java/io/quarkus/stork/StorkConfigUtil.java

```diff
@@ -15,10 +15,13 @@ public static List<ServiceConfig> toStorkServiceConfig(StorkConfiguration storkC
         SimpleServiceConfig.Builder builder = new SimpleServiceConfig.Builder();
         for (String serviceName : servicesConfigs) {
             builder.setServiceName(serviceName);
+
             ServiceConfiguration serviceConfiguration = storkConfiguration.serviceConfiguration.get(serviceName);
             SimpleServiceConfig.SimpleServiceDiscoveryConfig storkServiceDiscoveryConfig = new SimpleServiceConfig.SimpleServiceDiscoveryConfig(
                     serviceConfiguration.serviceDiscovery.type, serviceConfiguration.serviceDiscovery.params);
             builder.setServiceDiscovery(storkServiceDiscoveryConfig);
+            builder.setSecure(serviceConfiguration.serviceDiscovery.secure);
```

Comment:
I think this won't work with Stork 1.1.2, I'll update Quarkus to it this/next week and we'll see when we rebase the PR